setup.py

```diff
@@ -52,11 +52,12 @@
     packages=find_packages(),
     package_data={'pfio': package_data},
     extras_require={
-        'test': ['pytest', 'flake8', 'autopep8', 'parameterized', 'isort', 'moto[server]>=5.0.0', 'numpy', 'mypy'],
+        'test': ['pytest', 'flake8', 'autopep8', 'parameterized', 'isort', 'moto[server]>=5.0.0', 'numpy', 'mypy', 'pytorch-pfn-extras'],
```

Comment:
As we discussed offline, PPE shouldn't be here but tests must be switched in the test code using something like [skipif](https://docs.pytest.org/en/stable/how-to/skipping.html#id1).

Reply:
Changed to using `importorskip()` to skip if no ppe.

- no ppe
```
test_local.py ................s
```

- exist ppe
```
test_local.py .................
```

Reply:
We may also need to tweak the tox.ini to enable CI testing for the tracing code, which we're adding this PR if we remove PPE from the test section in setup.py: https://github.com/pfnet/pfio/blob/b874d493abb80ca7836f439fb7af37bac327b182/tox.ini#L4-L20

Reply:
Good catch